tasks/check-existence.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Get installed certificates.
+  shell: |
+    {{ certbot_script }} certificates | grep "Domains:" | awk '{ gsub(/    Domains: /,""); print }'
```

Comment:
This might only work on Linux. macOS, by default, ships with corresponding non-GNU programs, where some of them have a different behavior. It will also not work natively on Windows.

So, I believe this spot should be improved?

Reply:
Hi again. 9c0d000138 improves the situation by removing any bashisms.